src/services/global.ts

```diff
@@ -45,3 +31,13 @@ export const getSystemTheme = (): TTheme => {
     return "Light";
   }
 };
+// ---------------------------------
+export const getLineBreaker = (osType: TOsType): string => {
```

Comment:
Why do we even need this, if for all the known options it's gonna be "\n"

Reply:
sure, we can remove that.